timescaledb/how-to-guides/hypertables/create.md

```diff
@@ -74,9 +85,79 @@ hypertable fails. For more information about setting up multi-node, see the
      SELECT create_distributed_hypertable('conditions', 'time', 'location');
      ```
 
+<highlight type="note"> 
+If your table has a `PRIMARY KEY` or `UNIQUE` constraint, that key or constraint
+must include all columns that are used to partition your hypertable. In this
+example, the key or constraint must include `time` and `location`.
+
+It may additionally include columns that aren't partitioning columns. For more
+information, see [the troubleshooting
+section](https://docs.timescale.com/timescaledb/latest/how-to-guides/hypertables/create/#primary-keys-and-unique-constraints).
+</highlight>
+
 </procedure>
 
+## Troubleshooting
+
+### Primary keys and unique constraints
+
+If your `PRIMARY KEY` or `UNIQUE` constraint doesn't contain all your
+partitioning columns, you get an error. Partitioning columns always include the
```

Comment:
I'd avoid using the word include here, as it's used in the index context and could cause confusion. 

I might just make a brief separate section ie: 

Which columns are partitioning columns? 
1) The `time_column` used to create the hypertable- every hypertable partitions by its `time_column`. Therefore the `time_column` must be part of any `UNIQUE` constraint / `PRIMARY KEY`. 
2) Any space partitioning columns - space partitions are optional and not included on every hypertable, they may be defined with the `partitioning_columns` parameter of `create_hypertable`, or with the third
argument of `create_distributed_hypertable` - and I'd link to the API docs for those?